importccl: fix case sensitive INSERT INTO column names in PGDUMP

otan opened the pull request:
Refs: #52382 

Tested by uncommenting out the skip and re-commenting it afterwards.

Release note (sql change, bug fix): When using `IMPORT PGDUMP` with
`INSERT INTO` clauses, specifying a column name that is case sensitive
(e.g. `"cApItAls"`)  would previously error specifying the column name
was not found. This has been rectified.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/52485)
<!-- Reviewable:end -->

adityamaru reviewed (approved):
> Reviewed 2 of 2 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/52485)* status: :shipit: complete! 0 of 0 LGTMs obtained



<!-- Sent from Reviewable.io -->

otan commented:
thanks!

bors r=adityamaru

otan committed:
importccl: fix case sensitive INSERT INTO column names in PGDUMP

Tested by uncommenting out the skip and re-commenting it afterwards.

Release note (sql change, bug fix): When using `IMPORT PGDUMP` with
`INSERT INTO` clauses, specifying a column name that is case sensitive
(e.g. `"cApItAls"`)  would previously error specifying the column name
was not found. This has been rectified.

otan commented:
bors r=adityamaru
